src/libxtp/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ if(MKL_FOUND)
   target_link_libraries(votca_xtp PRIVATE MKL::MKL)
 endif()
 
+if(USE_CUDA)
+  target_link_libraries(votca_xtp PRIVATE ${CUDA_LIBRARIES} ${CUDA_CUBLAS_LIBRARIES})
```

Comment:
Change this to `PUBLIC` and it should be pulled in the unit tests automatically.

Reply:
I have done so, but if I removed the `CUDA_LIBRARIES` from the unit test, I get the following errors:
````
/usr/include/cublas_api.h(72): catastrophic error: cannot open source file "driver_types.h"
  #include "driver_types.h"
```

Reply:
Sorry, I left some dependencies as `PRIVATE`